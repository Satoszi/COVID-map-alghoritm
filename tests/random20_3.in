20 20 13 71
random20_3
..++++.|..-.++..+-.+
-+++-.++.-+-.+.+.|+|
.-.+.|.+.+-|++||+.++
+.+.-+..++.+..++.+.+
.|..-...+|+..|+++|-+
--+..+..+-.....|+-..
--+-+-.++.+.++....++
+....++.|++.+.|+++.+
.+++.-..+++|+.++..-.
++.++-++|.+.-++-..-.
|+-.++-+.|..+-|-++.|
.-.+.++-+.||+..+-+..
..++||-.+++++.+++|++
......+..+|+...+.+--
+-|.+-.++..+.+.++.-|
.++.+++..|++..-+...|
-...+..+.-..+-.-+|.+
++.+|+.-+.-|+.++..|+
.+--++..-+.+-..||..+
..+....++.|+.+.+.+|+
