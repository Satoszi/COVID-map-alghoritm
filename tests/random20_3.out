18 19
17 19
13 19
10 19
7 19
2 19
16 18
8 18
18 17
14 17
11 17
10 17
7 17
3 17
17 16
15 16
12 16
9 16
7 16
0 16
9 15
19 14
18 14
13 14
7 14
4 14
2 14
0 14
6 13
18 12
6 12
4 12
3 11
1 11
19 10
15 10
12 10
9 10
0 10
18 9
18 8
11 8
5 8
14 7
5 7
0 7
18 6
12 6
5 5
0 5
13 4
10 4
4 4
1 4
17 3
2 3
0 3
16 2
11 2
9 2
5 2
1 2
17 1
15 1
9 1
19 0
16 0
12 0
10 0
7 0
2 0
